src/pages/home/report/ReportActionsView.js

```diff
@@ -178,7 +178,20 @@ class ReportActionsView extends React.Component {
     updateSortedReportActions(reportActions) {
         this.sortedReportActions = _.chain(reportActions)
             .sortBy('sequenceNumber')
-            .filter(action => action.actionName === 'ADDCOMMENT' || action.actionName === 'IOU')
+            .filter((action) => {
+                // To filter deleted comments out we need to figure out if the message is empty
+                // (text or html equal to '')
+                // We first get the html and text from the action
+                // then we can check if the action has at least a non empty html or non empty text
+                const message = _.first(lodashGet(action, 'message', null));
+                const html = lodashGet(message, 'html', '');
+                const text = lodashGet(message, 'text', '');
+                return action.actionName === 'IOU'
+                    || (
+                        action.actionName === 'ADDCOMMENT'
+                        && !(html === '' && text === '')
```

Comment:
The code comment that was added says (twice) that either `html` or `text` can be empty but this logic checks that they are both empty. Which is the case?

Reply:
`!(html === '' && text === '')` is the same as `(html !== '' || text !== '' )` right?

Reply:
If it is the same, the second way you wrote that makes much more sense to me 😁 and matches the language in the comment better.

Reply:
lol, I'll change it back then, I had it originally like that, but @rory felt this way was better 😛 , I didn't felt strongly either way then, but now while reading the comment it makes more sense to go back to `(html !== '' || text !== '' )`.